cmd/kubeadm/app/cmd/init.go

```diff
@@ -240,31 +244,34 @@ func NewInit(cfgPath string, cfg *kubeadmapi.MasterConfiguration, ignorePrefligh
 	}
 
 	// Set defaults dynamically that the API group defaulting can't (by fetching information from the internet, looking up network interfaces, etc.)
+	glog.V(1).Infof("[init] Setting dynamic defaults")
 	err := configutil.SetInitDynamicDefaults(cfg)
 	if err != nil {
 		return nil, err
 	}
 
+	glog.V(1).Infof("[init] Validating kubernetes version")
 	if err := features.ValidateVersion(features.InitFeatureGates, cfg.FeatureGates, cfg.KubernetesVersion); err != nil {
 		return nil, err
 	}
 
-	fmt.Printf("[init] Using Kubernetes version: %s\n", cfg.KubernetesVersion)
-	fmt.Printf("[init] Using Authorization modes: %v\n", cfg.AuthorizationModes)
+	glog.Infof("[init] Using Kubernetes version: %s\n", cfg.KubernetesVersion)
+	glog.Infof("[init] Using Authorization modes: %v\n", cfg.AuthorizationModes)
 
 	// Warn about the limitations with the current cloudprovider solution.
 	if cfg.CloudProvider != "" {
-		fmt.Println("[init] WARNING: For cloudprovider integrations to work --cloud-provider must be set for all kubelets in the cluster.")
-		fmt.Println("\t(/etc/systemd/system/kubelet.service.d/10-kubeadm.conf should be edited for this purpose)")
+		glog.Warningln("[init] For cloudprovider integrations to work --cloud-provider must be set for all kubelets in the cluster.")
```

Comment:
Here you're terminating with a period `.`, but in other places you remove them. Can we pick one style and make that consistent?

Reply:
Sure.